packages/angular/src/lib/unpic.directive.ts

```diff
@@ -19,7 +25,15 @@ export class UnpicDirective {
 
   constructor(private el: ElementRef, private renderer: Renderer2) {}
 
+  ngOnChanges() {
+    this.syncProps();
+  }
+
   ngOnInit() {
+    this.syncProps();
+  }
```

Comment:
nit/suggestion: we could probably drop `ngOnInit` altogether because `ngOnChanges` also runs the first time when there are inputs.